### DISCUSSION TITLE
I can't get "npm run deploy" working

### DISCUSSION THREAD

**Discussion by Ph15** [velsa/notehost#33] [Q&A]
Hi there!

First of, many thanks for publishing this updated version of Fruition! 🙏

I've followed all the steps detailed by @DudeThatsErin on [NoteHost](https://dudethatserin.notion.site/NoteHost-982d31fcc8dd4799a18efcb074b0e63c), seemingly without any issue, but the last one resists.

I get the following error:

> `npm error Missing script: "start"
> npm error
> npm error Did you mean one of these?
> npm error   npm star # Mark your favorite packages
> npm error   npm stars # View packages marked as favorites
> npm error
> npm error To see a list of scripts, run:
> npm error   npm run
> 
> npm error A complete log of this run can be found in: /home/philipb/.npm/_logs/2024-05-09T12_40_44_099Z-debug-0.log`

Do you have any idea what it is about? (I have Googled the issue, but haven't found my way to a solution. I'm quite new to CLI and Linux by the way!)
Many thanks in advance for your help!
Kind regards,
— Philip

**Comment by DudeThatsErin** [comment]
When you run that what directory are you in? did the `npx wrangler login` command work?

**Reply by Ph15** [reply]
Oh, that was it! I didn't run the command from the right folder. Thanks!
I am now a step further, but with a new error:

> ✔ Would you like to continue? … yes
> Total Upload: 24.33 KiB / gzip: 7.07 KiB
> Uploaded vivrerelies-avpr-dir (2.75 sec)
> 
> ✘ [ERROR] A request to the Cloudflare API (/accounts/00b849972830e73426a40b72ec6f7861/workers/scripts/vivrerelies-avpr-dir/domains/records) failed.
> 
>   workers.api.error.origin_conflict_existing_dns_record [code: 100117]

Do you have any clue what that means? What kind of conflict could there be with an existing DNS record?

**Reply by DudeThatsErin** [reply]
Did you double check the worker’s name is correct in your files? Also, is the worker connected to your domain?

**Reply by Ph15** [reply]
Yep. The name was correct… but I hadn't updated my DNS! 🙈
I considered I had already done that step (since I already had a Fruition site running). But I still had active A records. So I delete them as advised, and everything seems to be working fine now! Thank you! 🙏

Is there any way to keep the search button active on the top bar? Or to add any other customization?
P.S.: I haven't managed to get the [favicon](https://archipelduvivant.org/wp-content/uploads/2021/05/vivre-relies.png) working (on [vivrerelies.org](vivrerelies.org)), but still much better than having a dysfunctional Fruition version. 😉

**Reply by Ph15** [reply]
Also, how can I add custom script in here?

>   // Custom JS for head and body of a Notion page
>   customHeadCSS: ``,
>   customHeadJS: googleTag(GOOGLE_TAG_ID),,
>   customBodyJS: PAGE_SCRIPT_JS_STRING,
> }

I would like to add Matomo analytics in the Head:

>   const CUSTOM_SCRIPT = `  
> <script>
>   var _paq = window._paq = window._paq || [];
>   /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
>   _paq.push(['trackPageView']);
>   _paq.push(['enableLinkTracking']);
>   (function() {
>     var u="//XXXXXXXXXX/";
>     _paq.push(['setTrackerUrl', u+'piwik.php']);
>     _paq.push(['setSiteId', '1']);
>     var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
>     g.async=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
>   })();
> </script>
>

**Reply by Ph15** [reply]
Well, in the end: it actually worked for some time, but on my main browser both websites show up as "NoteHost: Page Not found (404).". Oddly enough, both work on Brave. One out of two works on my phone. (All of this after having cleaned my cache and cookies, but it didn't help.)

Do you have any idea why?
I think I'll simply get back to a regular worker setup in the meantime. Thanks for paving new ways of building Notion-based websites anyway!
